### DISCUSSION TITLE
Old YML - Deprecated items

### DISCUSSION THREAD

**Discussion by RUHavingFun** [mrlt8/docker-wyze-bridge#252] [General]
I realized recently I had "network_mode: host" and "LAN_ONLY=True" in my .yml file.  I see the new "NET_MODE=LAN" option in a few comments and on the main page.  Is there an easy way to check the deprecations or see what has become what?  Just trying to keep current.  I also recently learned I could switch to using the latest image rather than build for the Raspberry Pi.  These are all good improvements but I was not immediately aware.  Maybe I need to review the change logs better, but sometimes I don't.

Also, I recently rebuilt my Pi's from scratch and found that the git code was no longer on the main page to show how to get the code pulled down originally to the Pi.  Why was "git clone https://github.com/mrlt8/docker-wyze-bridge.git" removed from the info page?  Should we be doing something else?  I understand it is kinda obvious for those familiar with git, but it was nice as I was fairly new my first time through this.

**Comment by mrlt8** [comment]
A deprecation warning should show up when starting up the container. This should stick around for each minor version 0.**X**.0

The git commands were removed in favor of using the pre-built images since you can essentially run the container with a [single command](https://github.com/mrlt8/docker-wyze-bridge#docker-run) on any of the supported architectures.
